app/bundles/FormBundle/Entity/Field.php

```diff
@@ -243,6 +248,7 @@ public static function loadMetadata(ORM\ClassMetadata $metadata): void
         $builder->addNullableField('alwaysDisplay', Types::BOOLEAN, 'always_display');
         $builder->addNullableField('mappedObject', Types::STRING, 'mapped_object');
         $builder->addNullableField('mappedField', Types::STRING, 'mapped_field');
+        $builder->addNullableField('fieldWidth', Types::STRING, 'field_width');
```

Comment:
```suggestion
        $builder->createField('fieldWidth', Types::STRING)
            ->columnName('field_width')
            ->length(50)
            ->option('default', 100)
            ->build();
```

The field definition in the entity must be the same as in the migration so there will be no diff when you run `bin:console doctrine:schema:generate --dump-sql` after the migration. I can see that in the entity you are missing the length of 50. We could even change it to like 3 if we'd use percentages. 6 if we'd use percentages with 2 decimal points. And here you are adding a nullable field and in the migration it's not nullable. I'd suggest to have it not nullable and add a default value of 100%.

Reply:
I'll just commit your suggestion, check if everything is right after